compiler/ras/Debug.cpp

```diff
@@ -3689,6 +3689,8 @@ TR_Debug::getRuntimeHelperName(int32_t index)
 
          case TR_newObject:                 return "jitNewObject";
          case TR_newObjectNoZeroInit:       return "jitNewObjectNoZeroInit";
+         case TR_newValue:                 return "jitNewObject";
+         case TR_newValueNoZeroInit:       return "jitNewObjectNoZeroInit";
```

Comment:
These returns are not aligned with the others (I think there's just one space missing).

Reply:
fixed